src/main/java/com/twilio/TwilioBearerTokenAuth.java

```diff
@@ -17,20 +20,34 @@ public class TwilioBearerTokenAuth {
     private static String region = System.getenv("TWILIO_REGION");
     private static String edge = System.getenv("TWILIO_EDGE");
     private static volatile BearerTokenTwilioRestClient restClient;
+    @Setter
+    private static TokenManager tokenManager;
 
     private static volatile ExecutorService executorService;
 
     private TwilioBearerTokenAuth() {
     }
 
-    public static synchronized void init(final String accessToken) {
-        if (accessToken == null || accessToken.isEmpty()) {
-            throw new AuthenticationException("Access Token can not be null or Empty");
+    public static synchronized void init(String grantType, String clientId, String clientSecret) {
+        validateAuthCredentials(grantType, clientId, clientSecret);
+        tokenManager = new OrgsTokenManager(grantType, clientId, clientSecret);
+    }
+    public static synchronized void init(String grantType, String clientId, String clientSecret, String code, String redirectUri, String audience, String refreshToken, String scope) {
+        validateAuthCredentials(grantType, clientId, clientSecret);
+        tokenManager = new OrgsTokenManager(grantType, clientId, clientSecret, code, redirectUri, audience, refreshToken, scope);
```

Comment:
Bug:
Follow below steps:
**Case1:**
1. CallOrgs API
If orgs api is called without init, then it will throw null pointer exception. This exception need to be handled.

**Case2:**
1. TwilioBearerTokenAuth.init() // Initialize
2. TwilioBearerTokenAuth.invalidate() // invalidate
3. CallOrgs API
In case2, previous invalidated token manager will be used, so during invalidate, tokenManager needs to be invalidated.